backend/src/firebase.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import { initializeApp } from "firebase/app";
+
+require("dotenv").config();
+
+const { FIREBASE_API_KEY } = process.env;
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: FIREBASE_API_KEY,
+  authDomain: "origins-digital.firebaseapp.com",
+  projectId: "origins-digital",
+  storageBucket: "origins-digital.appspot.com",
+  messagingSenderId: "898956315661",
+  appId: "1:898956315661:web:8d467e832ba230ef295944",
+};
```

Comment:
why do you need firebase ? and are you sure these ids can be seen into a public repository ?

Reply:
I need firebase to store the videos outside the backend as requested by the client and I have hidden the most important thing which is my API key

Reply:
even if storing outside the backend was mentionned in the product backlog, in the context of the training you will use the same server for storing the videos : so no need of firebase

Reply:
OK